app/Jobs/SendToBlink.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Illuminate\Bus\Queueable;
+use DoSomething\Gateway\Blink;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class SendToBlink implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * Blink API client.
+     *
+     * @var \DoSomething\Gateway\Blink
+     */
+    protected $blink;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Blink $blink)
```

Comment:
Oh! This is super unintuitive, but dependencies are injected as parameters of the `handle` method for jobs ([example](https://laravel.com/docs/5.5/queues#class-structure)). So the `Blink $blink` parameter would move down, and this line would take your job payload.

Reply:
omg I totally missed that!! Thank you!